sap_hana/README.md

```diff
@@ -0,0 +1,111 @@
+# Agent Check: SAP HANA
+
+## Overview
+
+This check monitors [SAP HANA][1] through the Datadog Agent.
+
+## Setup
+
+### Installation
+
+The SAP HANA check is included in the [Datadog Agent][2] package.
+
+#### Prepare HANA
+
+To query certain views, specific privileges must be granted to the chosen HANA monitoring user.
+
+##### User creation
+
+1. Connect to the system database and run the following command to create a user:
+
+```
+CREATE RESTRICTED USER <USER> PASSWORD <PASSWORD>
+```
+
+2. Run the following command to allow the user to connect to the system:
+
+```
+ALTER USER <USER> ENABLE CLIENT CONNECT
+```
+
+3. (optional) To avoid service interruption you may want to make the password long-lived:
+
+```
+ALTER USER <USER> DISABLE PASSWORD LIFETIME
+```
+
+##### Granting privileges
+
+1. Run the following command to create a monitoring role (we'll call it `DD_MONITOR` for these examples):
+
+```
+CREATE ROLE DD_MONITOR
+```
+
+2. Run the following command to grant read-only access to all system views:
+
+```
+GRANT CATALOG READ TO DD_MONITOR
+```
+
+3. Then run the following commands:
+
+```
+GRANT SELECT ON SYS.M_DATABASE TO DD_MONITOR
+GRANT SELECT ON SYS.M_DATABASES TO DD_MONITOR
+GRANT SELECT ON SYS_DATABASES.M_BACKUP_PROGRESS TO DD_MONITOR
+GRANT SELECT ON SYS_DATABASES.M_CONNECTIONS TO DD_MONITOR
+GRANT SELECT ON SYS_DATABASES.M_DISK_USAGE TO DD_MONITOR
+GRANT SELECT ON SYS_DATABASES.M_LICENSES TO DD_MONITOR
+GRANT SELECT ON SYS_DATABASES.M_RS_MEMORY TO DD_MONITOR
+GRANT SELECT ON SYS_DATABASES.M_SERVICE_COMPONENT_MEMORY TO DD_MONITOR
+GRANT SELECT ON SYS_DATABASES.M_SERVICE_MEMORY TO DD_MONITOR
+GRANT SELECT ON SYS_DATABASES.M_SERVICE_STATISTICS TO DD_MONITOR
+GRANT SELECT ON SYS_DATABASES.M_VOLUME_IO_TOTAL_STATISTICS TO DD_MONITOR
+```
+
+4. Finally, run the following command to assign the monitoring role to the desired user:
+
+```
+GRANT DD_MONITOR TO <USER>
+```
+
+### Configuration
+
+1. Edit the `sap_hana.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your sap_hana performance data. See the [sample sap_hana.d/conf.yaml][3] for all available configuration options.
+
+2. [Restart the Agent][4].
+
+### Validation
```

Comment:
Logs are available in `example.config.yaml`, but Log collection section in `readme.md` is missing.

Reply:
I'll remove logs for now, 👍